.github/workflows/ci-actions.yml

```diff
@@ -240,6 +231,51 @@ jobs:
           name: test-reports-linux-jvm${{matrix.java.name}}
           path: 'test-reports.tgz'
 
+  linux-gradle-jvm-tests:
+    name: JDK ${{matrix.java.name}} JVM Tests
```

Comment:
Should we mention Gradle in the name here?

Reply:
I updated it to :
````
linux-jvm-gradle-tests:

  name: JDK ${{matrix.java.name}} JVM Gradle Tests
````